.mbedignore

```diff
@@ -0,0 +1,12 @@
+boot/boot_serial/*
+boot/mynewt/*
+boot/zephyr/*
+boot/cypress/*
+ci/*
+docs/*
+ext/*
```

Comment:
@AGlass0fMilk 
TinyCrypt is ignored here (in `ext`), so it doesn't actually get built. Rather than ignoring the whole `ext`, please filter out its sub-directories instead.

After making TinyCrypt available (and change the signature algorithm to EC256, because RSA enables Mbed TLS no matter what crypto backend you specific in the config), I think you may get the issue I mentioned in https://github.com/mcu-tools/mcuboot/pull/791#discussion_r514984219

Reply:
And one more issue: If you compile with the Arm Compiler instead of GCC, `#include <version>` (e.g. in Mbed OS system code) will wrongly pick `ext/tinycrypt/VERSION` as `<version>` on case-insensitive file systems like those on Windows and macOS.

It's a known issue - the current Mbed build system recursively makes _every_ path an include path, including `ext/tinycrypt/` (but we only want `ext/tinycrypt/lib/include/`). There's no solution to this, until Mbed OS has officially announced CMake support which has fine-grained control over include paths - it's still in active development at the moment.

Reply:
So, to enable TinyCrypt for Mbed OS, we need to
* resolve https://github.com/mcu-tools/mcuboot/pull/791#discussion_r514984219
* restrict compilation to GCC only (and document this) until the new CMake-based build system is in place